tests/src/Unit/SectionsCollectorTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace {
```

Comment:
Creative way of mocking a filesystem 😆 
Not necessary for this PR, but perhaps we should refactor the finding/loading of templates to use Symfony's [Finder](https://symfony.com/doc/current/components/finder.html) and [FileSystem](https://symfony.com/doc/current/components/filesystem.html), so we can properly mock them.